inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -439,6 +439,12 @@ def _is_other_language(language):
         widget=defensedate_widget,
     )
 
+    defense_date = fields.TextField(
+        label=_('Date of Submission'),
```

Comment:
no, looks like wrong copy/paste. `thesis_date` is date of submission, `defense_date` is date of defense.